### PR TITLE
Minor fixes that I uncovered while testing.

### PR DESCRIPTION
**Pull request by jimcurtis**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -61,6 +61,7 @@ ansible_provision = proc do |ansible|
       ceph_mon_docker_subnet: "#{SUBNET}.0/24",
       ceph_osd_docker_extra_env: "CEPH_DAEMON=OSD_CEPH_DISK,OSD_JOURNAL_SIZE=100",
       ceph_osd_docker_devices: settings['disks'],
+      ceph_docker_on_openstack: OSVM,
```
Comment (leseb) [inline Vagrantfile]: We should not assume that we are running this on openstack by default.
So either you create a new condition for this or you update your `group_vars/all` file.

File: roles/ceph-osd/tasks/docker/main.yml [hunk 2]
```diff
@@ -20,11 +20,11 @@
 
 - include: pre_requisite.yml
 
+- include: dirs_permissions.yml
```
Comment (leseb) [inline roles/ceph-osd/tasks/docker/main.yml]: why?

**Comment by leseb** [comment]
Just fixed the docker image hardcoded path, please rebase.

File: Vagrantfile (hunk 1 above)
Comment (jimcurtis) [inline Vagrantfile]: The condition for this is at line #22

File: roles/ceph-osd/tasks/docker/main.yml (hunk 2 above)
Comment (jimcurtis) [inline roles/ceph-osd/tasks/docker/main.yml]: dirs_permissions.yml creates the directories that are need by fetch_configs.yml

**Pull request closed without merging**